api/views.py

```diff
@@ -1,6 +1,89 @@
 from django.http import HttpResponse
+from django.shortcuts import render
+from django.core.files.storage import FileSystemStorage
+from django.conf import settings
+import os
+import nltk
+import pytesseract
+import re
+import slate3k as slate
+import pdf2image
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize, sent_tokenize
+from nltk.stem.snowball import SnowballStemmer
+from PIL import Image
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract.exe'
 
 def consume_file(request):
     if request.method == 'POST':
         print(request.FILES['file'])
-        return HttpResponse(request.FILES['file'].name)
\ No newline at end of file
+        return HttpResponse(request.FILES['file'].name)
```

Comment:
Did you test this?

Reply:
Yes, and It has an error, I said to you before I dont know how to make It work like this I tried
![error](https://user-images.githubusercontent.com/108727944/204166308-59de004a-74b1-4c8d-87ca-55b6c13292a0.png)

Reply:
Check your code again. The return is at the wrong place.

Reply:
return HttpResponse(request.FILES['file'].name) This return at the end of file? I put It and It doesnt work neither

Reply:
![imagen_2022-11-30_172149850](https://user-images.githubusercontent.com/108727944/204852259-fb66cf1d-8df1-448e-be6e-14bbd052d279.png)
![imagen_2022-11-30_172216194](https://user-images.githubusercontent.com/108727944/204852362-06b79263-9605-4ee8-b3de-34137c5eb1a6.png)